modules/src/core/ics26_routing/handler.rs

```diff
@@ -386,12 +370,6 @@ mod tests {
                 msg: Ics26Envelope::Ics4ChannelMsg(ChannelMsg::ChannelOpenAck(msg_chan_ack)),
                 want_pass: true,
             },
-            //ICS20-04-packet
-            Test {
```

Comment:
These tests are disabled due to the new way we call into `ics20_msg_dispatcher`, which is done via `ics4_packet_callback`.